target/start-mailserver.sh

```diff
@@ -982,7 +982,7 @@ function _setup_postfix_aliases
       DOMAIN=$(echo "${FROM}" | cut -d @ -f2)
 
       # if they are equal it means the line looks like: "user1     other@domain.tld"
-      [ "${UNAME}" != "${DOMAIN}" ] && echo "${DOMAIN}" >> /tmp/vhost.tmp
+      [[ ${UNAME} != "${DOMAIN}" ]] && echo "${DOMAIN}" >>/tmp/vhost.tmp
```

Comment:
Quotes missing for ${UNAME}?

Reply:
That's an interesting question, but as far as I know, the answer is _no_:)

This is a Bash specific thing, were these string comparisons only need quotes on the **right** side. You know, when using the `[[ ]]` construct in Bash, you normally do not need quotes when checking a variable, e.g. `[[ -f ${FILE} ]]` is totally valid and the equivalent to `[ -f "${FILE}" ]` in POSIX `sh`. String comparisons are an exception to this rule, where you are best advised with using quotes on the right hand side (to prevent glob mathcing, see [this shellcheck entry](https://github.com/koalaman/shellcheck/wiki/SC2053)).

Reply:
Ack. But then we should keep it uniformly. There are some `[[ "` left in that file that should be changed too. I can provide a PR for that if you want.

Reply:
That would be awesome. I must have missed them:) Thanks!